.gitignore

```diff
@@ -27,3 +27,5 @@ yarn-error.log*
 !.yarn/sdks
 !.yarn/versions
 # .pnp.* # Comment for zero-install
+package.json
```

Comment:
This is odd, why git ignore the package list?

I understand, some may want to not store `package-lock.json`

Reply:
mistake on my part. fixed.